refinery/ui/source/js/data-set-import/controllers/file-upload-s3.spec.js

```diff
@@ -0,0 +1,269 @@
+(function () {
+  'use strict';
+
+  describe('RefineryFileUploadS3Ctrl', function () {
+    var $controller;
+    var $rootScope;
+    var s3UploadService;
+    var ctrl;
+    var scope;
+
+    beforeEach(module('refineryApp'));
+    beforeEach(module('refineryDataSetImport'));
+    beforeEach(inject(function (_$controller_, _$rootScope_, _s3UploadService_) {
```

Comment:
OK but I think I should still wrap `s3UploadService` because I would [have to use a different name for it locally in tests](https://stackoverflow.com/a/15318137) otherwise.